src/codesy_home.coffee

```diff
@@ -1,31 +1,26 @@
 # replace the install instructions with a check mark
-
 if $(".installed").length > 0
   $(".install-step").hide()
   $(".installed").show()
 
-console.time("codesy: check token");
 
-if $("#api_token_pass").length > 0
-  token_found = $("#api_token_pass").val()
-  if token_found isnt "" 
-    console.log("codesy: found token");
-    new_token = 
-      'domain': window.location.href
-      'token': token_found
+codesy={}
+codesy.find = (domains,domain) ->
+  domains.map((item) -> item.domain).indexOf(domain)
+
+
+console.time("codesy: token set");
+
+new_token = 
```

Comment:
A new domain is now added to local storage on every new page.  This allows the token to be cleared on sign out and eliminates all the if/then checks